Sources/RichTextKit/Bundle/Bundle+RichTextKit.swift

```diff
@@ -69,7 +69,7 @@ extension Bundle {
     }
 
     func bundlePath(for locale: Locale) -> String? {
-        bundlePath(named: locale.identifier) ?? bundlePath(named: locale.languageCode)
+        bundlePath(named: locale.identifier) ?? bundlePath(named: locale.language.languageCode?.identifier)
```

Comment:
@ropellanda I had to revert this, since it didn't build on macOS. Let me know if this is needed for it to work on visionOS.

Reply:
No, it's just noted as deprecated but still works ok, will open a pull request as soon as I fix it

Reply:
@danielsaidi check #124